lib/src/models/gigya_error.dart

```diff
@@ -20,7 +20,7 @@ class GigyaError implements Exception {
     }
 
     // Remove the specific error details, to avoid including them twice.
-    final int apiVersion = details.remove('apiVersion') != null ? details.remove('apiVersion') as int : 0;
+    final int? apiVersion = details.remove('apiVersion') as int?;
```

Comment:
This was taken from the recently merged PR in main. Calling `remove()` twice is a bug anyway, once it is removed, the second time it will return null.